src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -304,8 +303,13 @@ private void setDriverPropertyIfMissing(final String driverCommand, final String
         }
     }
 
+    private boolean isWindows() {
+        // Same as hudson.Functions.isWindows()
+        return File.pathSeparatorChar == ';';
+    }
```

Comment:
For consistency with the established convention in this repository, use `SystemUtils.IS_OS_WINDOWS`.